README.md

```diff
@@ -286,6 +286,14 @@ fastify.addHook('preHandler', function (request, reply, done) {
 ```
 Properties from `reply.locals` will override those from `defaultContext`, but not from `data` parameter provided to `reply.view(template, data)` function.
 
+To require `point-of-view` as a dependency to a [fastify-plugin](https://github.com/fastify/fastify-plugin) add the name `point-of-view` to the dependencies array in the [plugin's opts](https://github.com/fastify/fastify-plugin#dependencies).
```

Comment:
I understand the code's change, but I don't think the snippet is clear

I would just improve a bit:

```js
fastify.register(myViewRendererPlugin,   {
  dependencies: ['point-of-view']
})
```

Reply:
Totally. I've updated the README as per your suggestion.